staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -517,7 +526,7 @@ func (r Request) finalURLTemplate() url.URL {
 			segments[index+3] = "{name}"
 		}
 	}
-	url.Path = path.Join(segments...)
+	url.Path = strings.Join(segments, "/")
```

Comment:
@caesarxuchao can you double check the changes in this file?

Reply:
We should not be switching to strings.Join from path.Join without a really good reason.